### DISCUSSION TITLE
求程序代码讲解

### DISCUSSION THREAD

**Discussion by zhugexunyu** [peng-zhihui/ElectronBot#50] [Q&A] [answered]
- stm32F4的那块板子，程序初始化后，直接把唯一的任务给删除了，程序还能跑吗？
- 我也没有找到摄像头那部分的；
- 舵机驱动板的反驱没懂怎么做的，😵

**Comment by lVRuixiao** [comment]
对也，我也只看到了这个任务，其他的任务都没找到。
舵机实现反驱好像可以用编码器反馈脉冲，AB相又可以得到转向。如果用绝对式编码器就更稳妥了，直接把角度反馈回来。个人看法，理论应该可以。

**Comment by peng-zhihui** [comment, marked as answer]
* 任务初始化之后handle就没用了，跳转到main.cpp里面的Main()
* 摄像头翻一下close的issues，是usb-hub透传的
* 舵机里面都是电位器，ADC采样反馈

**Reply by zhugexunyu** [reply]
感谢大佬解惑，膜拜！！！

**Reply by zhugexunyu** [reply]
希望大佬各个程序能出个说明，不求多详细，大致画画流程(不敢浪费大佬时间)

**Comment by peng-zhihui** [comment]
找空会补充的。

**Comment by chenguoan001** [comment]
请问看那类的书籍可以掌握基本的知识？
